packages/client/src/components/config/AbstractMetricModal.tsx

```diff
@@ -430,6 +548,124 @@ export abstract class AbstractMetricModal<T extends CanaryMetricSetQueryConfig>
 
                 {/* Inject the metric source specific JSX here */}
                 {this.getMetricSourceSpecificJsx()}
+                <Accordion
+                    allowZeroExpanded={true}
+                    onChange={() => this.setState({showAdvancedConfiguration : !this.state.showAdvancedConfiguration})}
+                >
+                    <AccordionItem>
+                      <AccordionItemHeading>
+                        <AccordionItemButton>
+                          <div className="titled-section-title">Advanced<FontAwesomeIcon className="img chevron" size="sm" color="black" icon={this.state.showAdvancedConfiguration?faChevronCircleDown:faChevronCircleRight} /></div>
```

Comment:
Some additional whitespace could make this line more readable
For example:

> <div className="titled-section-title">
          Advanced<FontAwesomeIcon className="img chevron" size="sm" color="black" 
                 icon={this.state.showAdvancedConfiguration ? faChevronCircleDown : faChevronCircleRight} />
</div>

Reply:
`yarn format` took care of this